sites/public/src/components/shared/MapControl.module.scss

```diff
@@ -0,0 +1,32 @@
+$control-border-radius: 100px;
```

Comment:
can we use the border radius variable? https://github.com/metrotranscom/Doorway/blob/344459c7939201ef363f95239a42f1097bc4e3f1/doorway-ui-components/src/global/tokens/borders.scss#L9